src/state/middlewares/socket-io/chatlist.js

```diff
@@ -353,6 +355,18 @@ export default ( { io, timeout = 1000, customerDisconnectTimeout = 90000, custom
 		.catch( e => debug( 'failed to remove operator from chat', e.message ) )
 	}, chat_id => debug( 'chat.leave without existing chat', chat_id ) )( action.chat_id, action.user )
 
+	const handleCustomerBlock = action => whenChatExists( ( chat  ) => {
```

Comment:
damn it :( I installed the hooks but something was not right with `eslines` because it would through lots of errors even for code that I did not touch in the file.